flink-streaming-java/src/test/java/org/apache/flink/streaming/api/graph/StreamingJobGraphGeneratorTest.java

```diff
@@ -656,16 +658,17 @@ public void testNotSupportInputSelectableOperatorIfCheckpointing() {
 
 	@Test
 	public void testManagedMemoryFractionForSpecifiedResourceSpec() throws Exception {
+		// these specific values are needed to produce the double precision issues
```

Comment:
be more explicit what issues you are referring to

Reply:
updated.